### ISSUE TITLE
Support for Python 3.6? missing module dataclasses.

### ISSUE THREAD

**Issue by tru** [BlankSpruce/gersemi#2]
Hello,

I wanted to test your formatter on our codebase but we are currently running Python 3.6 and it's a bit complicated to roll out 3.7 everywhere. I changed the version in setup.py to make it install - but it fails with the following error:

```
Traceback (most recent call last):
  File "/home/tobias/.pyenv/versions/3.6.4/bin/gersemi", line 11, in <module>
    load_entry_point('gersemi', 'console_scripts', 'gersemi')()
  File "/home/tobias/.pyenv/versions/3.6.4/lib/python3.6/site-packages/pkg_resources/__init__.py", line 587, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/home/tobias/.pyenv/versions/3.6.4/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2800, in load_entry_point
    return ep.load()
  File "/home/tobias/.pyenv/versions/3.6.4/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2431, in load
    return self.resolve()
  File "/home/tobias/.pyenv/versions/3.6.4/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2437, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/home/tobias/code/gersemi/gersemi/__main__.py", line 5, in <module>
    from gersemi.configuration import (
  File "/home/tobias/code/gersemi/gersemi/configuration.py", line 2, in <module>
    from dataclasses import asdict, dataclass, fields
ModuleNotFoundError: No module named 'dataclasses'
```

I am unsure - but this doesn't seem to be a problem with the python version dependency? Is there a file missing in the repo or something else I missed. 

I installed it with `pip install -e .` from the checked out repo.

**Comment by BlankSpruce** [comment]
`dataclasses` library wasn't available before Python 3.6 since it's introduced with Python 3.7. I've never had a reason to maintain compatibility with Python 3.6 but I'll take a look if such change requires a lot of effort but no promises yet.

**Comment by BlankSpruce** [comment]
`New in version 3.7.`

https://docs.python.org/3/library/dataclasses.html

**Comment by BlankSpruce** [comment]
It turns out it was easy-peasy thanks to [dataclasses available in PyPI](https://pypi.org/project/dataclasses/). I've released version 0.5.0 so that you can pull out latest version from PyPI instead of using master. Let me know if you encounter some other issue.

As a side note it's possible to install any package from git repository like so:
`pip install git+https://github.com/BlankSpruce/gersemi.git`

**Comment by tru** [comment]
Thanks - works great!
